app/Http/Controllers/AddressController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\{Address,User,Area};
+use Illuminate\Support\Facades\Validator;
+
+class AddressController extends Controller
+{
+    public function userAddresses($id){
+        
+
+        $addresses = Address::where('user_id',$id)->get()->all();
```

Comment:
No need to use ->all()

Reply:
i need all() to check if array inside all is empty